app/Domain/Users/Templates/components/profile-image.blade.php

```diff
@@ -4,4 +4,5 @@
 
 <img {{ $attributes->merge([
     'src' => BASE_URL . '/api/users?profileImage=' . $user['id'] .'&v='.format($user['modified'])->timestamp(),
+    'style' => "display: inline;"
```

Comment:
Does this actually work across the board? (Check project hub). Display inline will prevent us from doing some repositioning.

Reply:
removed inline style. Changed style.default.css